src/Sulu/Bundle/AdminBundle/Resources/public/scss/modules/sticky-toolbar.scss

```diff
@@ -0,0 +1,31 @@
+.sticky-toolbar.fixed {
+    .list-toolbar-container {
+        width: 100%;
+        padding-right: 100px;
+        position: absolute;
+        background: #f1f1f1;
+        z-index: 98;
+        top: 30px;
+        padding-top: 10px;
+
+        &.fixed {
```

Comment:
fox example here https://github.com/sulu-io/sulu/pull/1821/files#diff-cd1787c96a81cb79e81a5d0ef80d3c2dR5 this will be used if the content has a fixed with.